MaterialDesignThemes.MahApps.nuspec

```diff
@@ -15,8 +15,8 @@
     <copyright>Copyright 2015 James Willock/Mulholland Software Ltd</copyright>
     <tags>WPF XAML WinRT Material Design Theme Colour Color UI UX</tags>
     <dependencies>
-      <dependency id="MaterialDesignColors" version="(1.1.3, 2.0)" />
-      <dependency id="MaterialDesignThemes" version="(2.5.1,3.0)" />
+      <dependency id="MaterialDesignColors" version="[1.2.0, 2.0)" />
+      <dependency id="MaterialDesignThemes" version="[2.5.1, 3.0)" />
```

Comment:
Can we change the lower bound of this to be `[2.6.0, 3.0)` please?

Reply:
Done